doc/tutorial_classification.rst

```diff
@@ -6,4 +6,165 @@
 Tutorial
 ========
 
-TO BE CONTINUED
\ No newline at end of file
+In this tutorial, we compare the prediction sets estimating by :class:`mapie.classification.MapieClassifier`.
+
+Throughout this tutorial, we will answer the following questions:
+
+- How do the number of classes in the prediction sets vary according to the significance level alpha values and the conformity scores ?
+
+1. Conformal Prediction method using the softmax score of the true label
+========================================================================
+We will use MAPIE to estimate a prediction set of several classes such that the probability that the true label of a new test point is included in the prediction set is always higher than the target confidence level 1−alpha.
```

Comment:
Mettre en equation le 1-alpha : :math:`1 - \alpha`

Reply:
Je rajouterais encore une petite phrase du style : "We start by using the softmax score output by the base classifier as the conformity score on a toy two-dimensional dataset."

Reply:
ajouté